enter_dev_env.sh

```diff
@@ -16,14 +16,14 @@ fi
 if [ -n "$HASH" ];then
     echo "found existing running container $CONTAINER, proceeding to exec another shell"
     docker-compose -f $COMPOSE_FILE restart
-    docker exec -it $HASH $START_SHELL
+    docker exec -w /app/src -it $HASH bash -c "SHELL=bash poetry shell"
```

Comment:
nit. we could also remove the variable `START_SHELL` if we specify the used SHELL here.